publish.js

```diff
@@ -43,7 +43,7 @@ shell.rm("-rf", "./dist/native/target");
 
 
 shell.pushd("./native");
-shell.exec("cargo update");
+// shell.exec("cargo update");
```

Comment:
Since develop branch of holochain-rust is such a fast moving Target right now, I wanted to stabilize this , and then uncomment it once we merged this, in another PR, where the fixes for latest develop can happen